example-app/src/app/components/camera-modal/camera-modal.component.ts

```
@@ -0,0 +1,671 @@
+  protected readonly isRecording = signal(false);
+  protected readonly currentOpacity = signal(100);
+  protected readonly testResults = signal<string>('');
+  protected readonly showTestResults = signal(false);
+
+  // Camera switching functionality
+  protected readonly availableCameras = signal<CameraDevice[]>([]);
+  protected readonly selectedCameraIndex = signal<number>(0);
+
+  protected readonly canZoomIn = computed(() => {
+    return this.currentZoomFactor() + 0.1 <= this.maxZoom();
+    return this.currentZoomFactor() + 0.1 <= this.maxZoom();
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)